### PR TITLE
Create Project Success metrics

### PR DESCRIPTION
**Pull request by jmertic**
Continued from https://github.com/AcademySoftwareFoundation/tac/pull/93

@mpgjon and @dheckenberg please comment on next steps

### CODE REVIEW

**Comment by mpgjon** [comment]
@jmertic - I might suggest the metrics for project success be derived from committee meetings of the individual project leads/founders and then baked into a subset that can be agreed between the projects.  

I can post some example metrics in the MD for discussion/review.

**Comment by mpgjon** [comment]
@jmertic - I don't have write access here, so it doesn't look like I can create a new file for review and merge here - only in my fork: https://github.com/mpgjon/aswf-tac/blob/master/process/success_metrics.md 

Take a look and if that's useful and I can get added to the team here, I'm happy to contribute.

**Comment by jmertic** [comment]
Thank you - added your input now @mpgjon. Let me know if you think this is ready for a more broad TAC discussion.

File: process/success_metrics.md [hunk 1]
```diff
@@ -118,7 +122,16 @@ As a project developer or contributor, I care most about:
...
++ How many users of the software do we have?
++ How many bugs do we have?
++ How long does it take for a bug to be fixed?
++ How long are PR's sitting open?
+
 ## Tools for measursing Project Success
 
-https://bitergia.com/ (commercial product)
-https://github.com/chaoss/ (open source product that Bitergia is based on)
+All projects have thier metrics posted on [LFX Insights](https://insights-v2.lfx.linuxfoundation.org/aswf/trends).
```
Comment (jfpanisset) [inline process/success_metrics.md:137]: thier -> their

**Comment by jfpanisset** [comment]
Looks like DCO is missing from commit https://github.com/AcademySoftwareFoundation/tac/pull/155/commits/bb9d05a400b0f60cea8f023802bb28a83ee41da2

**Comment by jmertic** [comment]
Good catch - I can squash the commits at the end.

**Pull request closed without merging**